Dockerfile

```diff
@@ -1,13 +1,22 @@
-FROM node:latest
+FROM python:3-alpine
 
-WORKDIR /app
+RUN mkdir -p /beehive-order-generator
 
-COPY package.json .
+ARG google_api_key
+ARG order_queue
+ARG rabbitmq
+ARG database
 
-RUN npm install --production
+ENV GOOGLE_API_KEY=$google_api_key
+ENV ORDER_QUEUE=$order_queue
+ENV RABBITMQ_URL=$rabbitmq
+ENV DATABASE_URL=$database
 
-COPY ./lib .
+COPY /src /beehive-order-generator/src
+COPY requirements.txt /beehive-order-generator
 
-RUN ls
+RUN pip3 install -r beehive-order-generator/requirements.txt
```

Comment:
copy only the requirements.txt file as the first step in the dockerfile and execute the install command, so that the dependencies won't be installed everytime code changes